README.md

```diff
@@ -157,3 +157,50 @@ Common usage:
   ```shell
   docker run --rm -v ~/solid-config:/config -p 3000:3000 -it css:latest -c /config/my-config.json
   ```
+
+## Using the identity provider
+
+1. Launch the Community Solid Server, for example in file server mode:
+    ```bash
+    git clone git@github.com:solid/community-server.git
+    cd community-server
+    mkdir data
+    npm ci
+    npm run start -- -c config/config-file.json -f ./data
+    ```
+2. To use the identity provider, you need a compatible client application.
+
+    You can use for example `@inrupt/solid-client-authn-js`:
+
+    ```bash
+    git clone https://github.com/inrupt/solid-client-authn-js
+    cd solid-client-authn-js
+    npm ci
+    cd packages/node/example/demoClientApp/
+    npm ci
+    npm run start
+    ```
+
+    Go to `http://localhost:3001`.
+3. Use the base URL of your running CSS instance to as Identity provider, for
+   example `http://localhost:3000`, to fill the form. Click the `login` button.
+4. Follow the instructions to register/login/...
+
+    A WebID hosted in your pod will be required to complete registration.
+    
+    In your community server running with a file backend as described in step 1,
+    you could create `./data/profile/card$.ttl` with the following content:
+    ```turtle
```

Comment:
I would change this part to not have the `solid:oidcIssuerRegistrationToken`. But add that during the registration process the IdP will ask to add a triple to your profile which you can do by PATCHing with (or overwriting everything with a PUT).
```sparql
PREFIX : <#>
PREFIX solid: <http://www.w3.org/ns/solid/terms#>
INSERT DATA { :me solid:oidcIssuerRegistrationToken "TOKEN_GIVEN_DURING_REGISTRATION_PROCESS" }
```

Reply:
Updated, let me know what you think ;)